### PR TITLE
Add `napari/plugins` and `utils/context` to typechecks

### PR DESCRIPTION
**Pull request by tlambert03**
# Description
fixes a few minor typing issues in `napari/plugins` and adds it to the type-checked modules

## Type of change

- [ ] Bug-fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# References

# How has this been tested?

- [ ] example: the test suite for my feature covers cases x, y, and z
- [ ] example: all tests pass with my change

## Final checklist:
- [ ] My PR is the minimum possible work for the desired functionality
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I included new strings, I have used `trans.` to make them localizable.
      For more information see our [translations guide](https://napari.org/docs/dev/guides/translations.html).

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -152,3 +153,4 @@ docs/guides/preferences.md
 napari/settings/napari.schema.json
 docs/guides/_layer_events.md
 docs/guides/_viewer_events.md
+napari/plugins/_builtins.py
```
Comment (tlambert03) [inline .gitignore]:
```suggestion
```

File: napari/plugins/_builtins.py [hunk 2]
```diff
@@ -39,9 +39,11 @@ def csv_reader_function(path: Union[str, List[str]]) -> List[LayerData]:
...
 
 
-def npy_to_layer_data(path: str) -> List[LayerData]:
-    data = np.load(path)
-    return [(data,)]
+def npy_to_layer_data(path: Union[str, Sequence[str]]) -> List[LayerData]:
+    if isinstance(path, str):
+        return [(np.load(path),)]
+
+    return [np.load(p) for p in path]
```
Comment (tlambert03) [inline napari/plugins/_builtins.py]:
```suggestion
    return [(np.load(p),) for p in path]
```

**Review by goanpeca — APPROVED** [review]
Wohoo, awesome @tlambert03 🚀

**Comment by goanpeca** [comment]
Hi @tlambert03 is there something like typing "coverage" so we can track it :) ?

**Comment by tlambert03** [comment]
Not that I know of

**Pull request merged**